src/CMakeLists.txt

```diff
@@ -14,8 +14,13 @@ include(CalamaresAddTranslations)
 add_subdirectory(libcalamares)
 add_subdirectory(libcalamaresui)
 
-# all things qml
-add_subdirectory(qml/calamares)
+if(WITH_QT6)
+  # all things qml
+  add_subdirectory(qml/calamares-qt5)
```

Comment:
Is this meant to be for WITH_QT6?  Seems this is for Qt5

Reply:
yes, fixed